### DISCUSSION TITLE
lengte

### DISCUSSION THREAD

**Discussion by giscus** [Roald87/roaldin.ch#30] [Announcements]
# lengte

Nederlandse mannen zijn met 1.838 m een van de langste mensen ter wereld. Maar als je in Nederland opgroeit, valt dat niet echt op, omdat je zelf deel uitmaakt van de statistiek. Althans, zo ervaarde ik het. Ik merkte pas hoe lang Nederlanders zijn toen ik hier kwam wonen. Zoals op het werk, waar ik met 1.85 m een van de langere ben. Of recent tijdens een concert. Geen verkeerde plek overigens om er letterlijk bovenuit te steken. Vriendin I. is op haar werk ook een van de langere, ondanks dat ze voornamelijk mannelijke collega’s heeft. Zij is met 1.78 m even groot als de doorsnee Zwitserse man. De meeste van haar mannelijke collega’s zitten dus onder het gemiddelde.

https://roaldin.ch/lengte

**Comment by oom-wim** [comment]
Geniet nog even van het uitzicht, de krimp zet in rond je 40ste. Dan heb ik het nog niet over de dagelijkse krimp van 1 %, maar daar herstel je 's nachts weer van. Ik heb ook 1.85 in mijn pas staan, maar dat klopt niet meer. Het is niet zo erg om te krimpen; toen er een patatduif in mijn gezicht vloog, had ik best wat kleiner willen zijn.

**Reply by Roald87** [reply]
Ja dat zie ik ook bij mijn vader. Ooit was hij even groot als J. en ik.

De patatduif :'), die kende ik nog niet.
